src/core_plugins/tile_map/public/maps_visualization.js

```diff
@@ -75,10 +79,13 @@ export function MapsVisualizationProvider(serviceSettings, Notifier, getAppState
       const centerFromUIState = uiState.get('mapCenter');
       options.zoom = !isNaN(zoomFromUiState) ? zoomFromUiState : this.vis.type.visConfig.defaults.mapZoom;
       options.center = centerFromUIState ? centerFromUIState : this.vis.type.visConfig.defaults.mapCenter;
+      options.isFilteredByCollar = this._isFilteredByCollar.bind(this);
```

Comment:
I don't think we need this functional approach. Also, this breaks the Region Map, because this option isn't applied  See other comments on moving this to `geohash_layer` for different approach.